autokeras/tuners/greedy.py

```diff
@@ -200,14 +204,14 @@ def _generate_hp_values(self, hp_names):
 class Greedy(tuner_module.AutoTuner):
     def __init__(
         self,
-        hypermodel,
-        objective="val_loss",
-        max_trials=10,
-        initial_hps=None,
-        seed=None,
-        hyperparameters=None,
-        tune_new_entries=True,
-        allow_new_entries=True,
+        hypermodel: kerastuner.HyperModel,
+        objective: str = "val_loss",
+        max_trials: int = 10,
+        initial_hps: Optional[List[Dict[str, Any]]] = None,
```

Comment:
@haifeng-jin I was not sure on the type for initial_hps the list of dict has str as Key but was able to take Any (bool, float, int, str) as value. Hope you are find with this.

Reply:
It think it looks good for now at least. If we have other use cases, we can change it later. Thanks.